src/js/pages/Department.js

```diff
@@ -29,11 +28,11 @@ const Department = ({ department }) => {
           <div className="coa-main__left col-xs-12 col-lg-8">
 
             <div className="coa-section">
-              <SectionTitle title={title}/>
+              <SectionHeader title={title}/>
             </div>
 
             <div className="coa-section">
-              <SectionTitle title="Our Mission"/>
+              <SectionHeader title="Our Mission"/>
```

Comment:
should we go ahead and put this text in some i18nMessage object?

Reply:
prolly, I haven't cared much about dept pages -- so I figured I'd do that work later as that page gets more stable.